cloud/main.js

```diff
@@ -84,28 +86,37 @@ function sendToFeedbackChannel(body, response) {
 //   "endTimeToken": "00000"}
 
 Parse.Cloud.define("replayEvents", function(request, response) {
-    var requiredParams = ["aliasId", "subkey"];
+    var requiredParams = ["aliasId"];
     var params = request.params;
     checkMissingParams(params, requiredParams, response);
 
-    var count = params.count ? params.count : 9999;
-    var aliasId = params.aliasId;
+    var count = params.count || 20;
     var subkey = params.subkey;
+    var startTimeToken = params.startTimeToken ?
+        new Moment(params.startTimeToken).toDate() : new Moment().toDate();
+    var endTimeToken = params.endTimeToken ? new Moment(params.endTimeToken).toDate() : undefined;
+    var aliasId = params.aliasId;
```

Comment:
`const { aliasId } = params;` is another way to do this
Plus you can do default values e.g. 
`const { aliasId = 'hey' } = params;`
this is destructing